code/modules/mining/materials.dm

```diff
@@ -137,6 +137,7 @@ var/global/list/initial_materials	//Stores all the matids = 0 in helping New
 	var/sharpness_mod = 1
 	var/quality_mod = 1
 	var/melt_temperature
+	var/armor_mod = 1
```

Comment:
I assume this is planned to be used in the future?

Reply:
Yes